goerli/README.md

```diff
@@ -0,0 +1,185 @@
+## Installation of EigenDA using docker
+
+### Prerequisites:
+* Docker installed
+* [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli) installed
+
+> NOTE: For any Docker based commands, if you have installed as root then you might have to append `sudo` in front of the command.
+
+### Core operations
+* Register your operator to EigenLayer using [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/README.md)
+
+### Setup EigenDA
+The easiest way to set up EigenDA is to clone the repo and follow the instructions below.
+
+#### Clone repo
+Clone this repo and execute the following commands:
+```bash
+git clone https://github.com/Layr-Labs/eigenda-operator-setup.git
+cd eigenda-operator-setup/goerli/
+cp .env.example .env
+```
+Update the `TODO` sections in the  `.env` file given in the root directory of the repository with your own details.:
+
+### Create some local folders which are required by EigenDA
+```bash
+mkdir -p $HOME/.eigenlayer/eigenda/logs
+mkdir -p $HOME/.eigenlayer/eigenda/db
+```
+
+### Operator Networking Security Setup
+Retrieval Setup:
+
+In order for users to retrieve data from your node, you will need to open access to retrieval ports. 
+
+Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/goerli/.env.example#L17) has open access to the public internet.
+
+For users with private IP e.g. connecting to the Internet via a router, you may need to perform port forwarding to open the retrieval ports. 
+Use a web browser and navigate to http://192.168.0.1 and set-up port forwarding according to instruction of your router.
+
+Dispersal Setup:
+
+In order to limit traffic from the EigenLabs hosted Disperser, please restrict your node's ingress traffic to be allowed by the list provided below and port number set as `NODE_DISPERSAL_PORT` in the [.env](./.env.example) in the below setup.
+
+* `3.221.120.68/32`
+* `52.2.226.152/32`
+* `18.214.113.214/32`
+
+For users with private IP, you may need to perform port forwarding to open the dispersal ports. Refer to retrieval setup for more details.
+
+### Opt-in into EigenDA
+This command also downloads the latest SRS points (~8 GB) if they don't exist and can take upto 10 minutes to complete for the first time based on your network speed.
```

Comment:
This may need to further refactor since SRS can be shared across environments so operator can download just once. There may be some shared pieces across environments. (could follow up separately)

Reply:
ya I mean if they have separate machines for mainnet and holesky and they are following this guide then they might have to download it again.

Reply:
will follow up separately